### PR TITLE
fix: update style of buttons in toast

### PR DESCRIPTION
**Pull request by abutterworth**
Update toast spacing, alignment, and button style.
image

Update icon buttons to have transparent backgrounds by default. This is configurable via the new SASS vars:
```
$icon-button-bg: transparent !default;
$icon-button-invert-bg: transparent !default;
```

### CODE REVIEW

File: src/IconButton/_IconButton.scss [hunk 1]
```diff
@@ -13,17 +13,21 @@ $icon-button-margin: .75rem !default;
 $icon-button-refwrapper-size: $icon-button-hover-size + $icon-button-margin !default;
 
 $icon-color-invert: white !default;
+$icon-button-bg: transparent !default;
+$icon-button-invert-bg: transparent !default;
```
Comment (abutterworth) [inline src/IconButton/_IconButton.scss:17]: New sass options for the background of icon buttons.

File: src/IconButton/_IconButton.scss [hunk 2]
```diff
@@ -52,12 +56,3 @@ $icon-color-invert: white !default;
...
 }
-
-.iconbutton-refwrapper {
-    height: $icon-button-refwrapper-size;
-    width: $icon-button-refwrapper-size;
-    padding: $icon-button-margin * 0.5;
-    button {
-        border: none;
-    }
-}
```
Comment (abutterworth) [inline src/IconButton/_IconButton.scss:63]: Removing this rewrapper until we find we need it again. It was kind of in the way of passing through custom classNames on the icon.

File: src/IconButton/index.jsx [hunk 3]
```diff
@@ -10,25 +11,30 @@ const IconButton = ({
   iconClassNames,
   onClick,
   variant,
+  ...attrs
```
Comment (abutterworth) [inline src/IconButton/index.jsx:14]: Pass in any html attributes, including className

File: src/Toast/_Toast.scss [hunk 4]
```diff
@@ -1,44 +1,34 @@
...
+  padding: 1.25rem;
   position: relative;
-  radius: $toast-border-radius;
+  border-radius: $toast-border-radius;
   z-index: 2;
-
+  .toast-header-btn-container {
+    margin: -.25rem -.5rem;
+    align-self: flex-start;
+  }
```
Comment (abutterworth) [inline src/Toast/_Toast.scss:13]: Added this contain to help with alignment of close button

**Review by ciduarte — APPROVED** [review]
Looks good! Thanks again

**Review by raq929 — APPROVED** [review]
LGTM!

**Pull request merged**